run/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+file(GLOB_RECURSE data_src "data/*")
+
+if(WIN32)
+    # Ensure that r2pkg can find its dependent DLLs
+    set(SETENV_FOR_UPDATE_DATA
+            ${CMAKE_COMMAND} -E env
+            "PATH=${REAPING2_DEPS_INSTALL_DIR}/bin$<SEMICOLON>$<TARGET_FILE_DIR:r2pkg>")
+endif(WIN32)
+
+add_custom_command(OUTPUT data.pkg
+        COMMAND
+            ${SETENV_FOR_UPDATE_DATA}
+            r2pkg -c ${CMAKE_CURRENT_SOURCE_DIR}/data
```

Comment:
Using the executable target name as the command will tells cmake to replace the call with the location of the executable created during build time, and also to create a target-level dependency (so r2pkg would be always built before the custom command runs).

True, this might not be the case on Windows, where the first command is the `${CMAKE_COMMAND}` and not the target name, but in that case using the `$<TARGET_FILE_DIR:r2pkg>` generator expression in the arguments has the same effect (creates a target level dependency).

Additionally, `find_program` would run during configuration time, when the r2pkg might not be built yet. The only use case I imagine for that would be if you wanted to use an independently built/installed r2pkg to run the update target